test/prefer-includes.js

```diff
@@ -0,0 +1,64 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/prefer-includes';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const errors = [{
+	ruleId: 'prefer-includes',
+	message: 'Use `.includes()`, rather than `.indexOf()`, when checking for existence.'
+}];
+
+ruleTester.run('prefer-includes', rule, {
+	valid: [
+		'str.indexOf(\'foo\') !== -n',
+		'str.indexOf(\'foo\') !== 1',
+		'!str.indexOf(\'foo\') === 1',
+		'!str.indexOf(\'foo\') === -n',
+		'str.includes(\'foo\')',
+		'\'foobar\'.includes(\'foo\')',
+		'[1,2,3].includes(4)',
+		'null.indexOf(\'foo\')'
+	],
+	invalid: [
+		{
+			code: '\'foobar\'.indexOf(\'foo\') !== -1',
+			output: '!\'foobar\'.includes(\'foo\')',
```

Comment:
You fixer changes the result of the expression:

![image](https://user-images.githubusercontent.com/4330357/51924678-8b76e100-23fe-11e9-8cc5-0b65b2d768ea.png)

Reply:
ops, I mistook.
Now it have to work fine.